.env-example

```diff
@@ -3,8 +3,6 @@
 NGINX_VERSION=1.23.3
 
 # PostgreSQL data
-POSTGRES_PASSWORD=Letmin123
```

Comment:
Why has `POSTGRES_PASSWORD` been removed? This was used to set the default `postgres` root password on the PostgreSQL instance?